DEV_README.md

```diff
@@ -0,0 +1,26 @@
+# Skadi Chart
+
+Simple d3 graphing library used by Skadi (Static [Wodin](https://github.com/smrc-ide/wodin))
+
+# Basic commands
+
+* `npm run build` - build project
+* `npm run test:unit` - unit tests
+* `npm run test:e2e` - end to end tests
+* `npm run coverage` - coverage with v8 as provider
+* `npm run dev` - start demo app in [./src/demo](./src/demo/)
+
+# Coordinate systems
+
+Some layers in this project deal with multiple coordinate systems and the code got quite confusing as we were not sure about which coordinate system certain variables were in. So we have tried to standardise some variables names to avoid confusion in some layer such as the [TooltipsLayer](./src/layers/TooltipsLayer.ts). We deal with 3 coordinate systems:
+
+![coordinate systems](./assets/coords.png)
+
+Each coordinate system has its origin highlighted with the circle, and x and y axes shown. Details for each color:
+
+* Red: These are the client coordinates that keep track of where elements in the DOM are relative to the red origin in the top left hand corner of the webpage. These coordinates are measured in pixels. Variables in this coordinate system will be suffixed with `CC` (client coordinates).
+
+* Green: The svg we generate has a view box equal to the width and height of the svg element, which means we have a 1 to 1 mapping of svg coordinates and client pixels (see [this](https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorials/SVG_from_scratch/Positions) for more details). The green axes then represent the position of elements of the svg relative to its top left corner. These are also measured in pixels. Variables in this coordinate system will be suffixed with `SC` (svg coordinates).
```

Comment:
```suggestion
* Green: The svg we generate has a view box equal to the width and height of the svg element, which allows us to have a 1-to-1 mapping of svg coordinates and client pixels (see [this](https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorials/SVG_from_scratch/Positions) for more details). The green axes then represent the position of elements of the svg relative to its top left corner. These are also measured in pixels. Variables in this coordinate system will be suffixed with `SC` (svg coordinates).
```

just slightly better at cuing me the reader to understand that this is a design decision, not something that would happen by default